Healthy/Classes/Extensions/UIButton+Style.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  UIButton+Style.swift
+//  Healthy
+//
+//  Created by Ahmed Nasr on 13/05/2023.
+//
+
+import UIKit
+
+
+//MARK: - button style -
+//
+extension UIButton {
+    enum buttonStyle {
```

Comment:
enums should start with upper case letter
```suggestion
    enum ButtonStyle {
```

Reply:
Done, Thanks Yasser 🚀👍